java/src/jmri/jmrix/loconet/LocoNetSystemConnectionMemo.java

```diff
@@ -274,8 +274,10 @@ public void configureManagers() {
 
         setConsistManager(new LocoNetConsistManager(this));
 
-        InstanceManager.addClockControl(
-                getClockControl());
+        ClockControl cc = getClockControl();
+        // make sure InstanceManager knows about that
+        InstanceManager.store(cc, ClockControl.class);
```

Comment:
Is there a need to call both store() and setDefault()? I thought this could be one line as `InstanceManager.setDefault(ClockControl.class, getClockControl());` since setDefault() calls store internally if needed.

Reply:
Thanks for catching that, updated.